src/software/ai/hl/stp/play/shoot_or_pass/shoot_or_pass_play_fsm.cpp

```diff
@@ -71,6 +71,7 @@ void ShootOrPassPlayFSM::lookForPass(const Update& event)
                                                    ->value();
         pass_eval = pass_generator.generatePassEvaluation(event.common.world);
         best_pass_and_score_so_far = pass_eval.getBestPassOnField();
+        std::cout << "SCORE IS: " << best_pass_and_score_so_far.rating << std::endl;
```

Comment:
Nit: this looks to be a debugging message, so it should be removed before merging